docs/core/whats-new/dotnet-9/libraries.md

```diff
@@ -27,18 +27,19 @@ The collection types in .NET gain the following updates for .NET 9:
 - [Collection lookups with spans](#collection-lookups-with-spans)
 - [`OrderedDictionary<TKey, TValue>`](#ordereddictionarytkey-tvalue)
 - [PriorityQueue.Remove() method](#priorityqueueremove-method) lets you update the priority of an item in the queue.
+- [`ReadOnlySet<T>`](#readonlysett)
 
 ### Collection lookups with spans
 
-In high-performance code, spans are often used to avoid allocating strings unnecessarily, and lookup tables with types like <xref:System.Collections.Generic.Dictionary%602> and <xref:System.Collections.Generic.HashSet%601> are frequently used as caches. However, it's been challenging to use these types together, as there was no safe, built-in mechanism for doing lookups on these types with spans. Now with the new `allows ref struct` feature in C# 13 and new features on these collection types in .NET 9, it's possible to perform these kinds of lookups.
+In high-performance code, spans are often used to avoid allocating strings unnecessarily, and lookup tables with types like <xref:System.Collections.Generic.Dictionary%602> and <xref:System.Collections.Generic.HashSet%601> are frequently used as caches. However, there has been no safe, built-in mechanism for doing lookups on these collection types with spans. With the new `allows ref struct` feature in C# 13 and new features on these collection types in .NET 9, it's now possible to perform these kinds of lookups.
 
-The following example demonstrates using `Dictionary<TKey, TValue>.GetAlternateLookup` <!--<xref:System.Collections.Generic.Dictionary%602.GetAlternateLookup?displayProperty=nameWithType>-->.
```

Comment:
@carlossanlop See these diffs? This suggests to me that @gewarren is taking a two step approach because API docs are lagging. If we could generate API docs sooner in the process (for next time), this wouldn't be needed.

Is that a fair analysis @gewarren?

Reply:
@richlander I'm in charge of the API docs publishing, not Carlos. He's mostly concerned with whether they're *documented with comments*. Does it make sense to publish the updated APIs before the release?

Reply:
Got it.

I see three options:

- Status quo
- Do everything in one branch
- Publish API docs early

I don't see status quo as serving anyone's needs. It seems like it is creating a bit of extra work for you and the API links lag significantly. That's not a complaint, just an observation. The ideal solution seems to be to do everything in one branch. That would fix it, right?

Reply:
> One branch
That wouldn't work because API docs and conceptual docs are in different repos.

Reply:
I see and also forgot.

I think it is worth trying an experiment. Ship the API docs one week early and then use that as the baseline for these docs. If people get confused / complain, we can adapt.

Thoughts? We can also ask others for feedback.

Reply:
Day of.

OK. Unless we change something about the consumption model, then the idea won't work.

Anyway, was just an idea.  There is no dramatic problem here. I was just wondering if there was an optimization opportunity. There may not be.